cli/app.go

```diff
@@ -2048,6 +2049,10 @@ Example:
 									Usage: "use this to indicate that your meta.json is in a subdirectory of your repo." + " --module flag should be relative to this",
 									Value: ".",
 								},
+								&cli.StringFlag{
+									Name:  moduleBuildFlagPlatforms,
+									Usage: "comma-separated list of platforms to build, e.g. linux/amd64,linux/arm64. Defaults to build.arch in meta.json.",
+								},
```

Comment:
(q) curious why we're asking for a comma-separated list rather than having this as a `StringSliceFlag`?

Reply:
didn't know about StringSliceFlag, very handy

fixed in 1660ebbea

tested this -- seems like StringSliceFlag accepts both of:
```sh
go run ./whatever --platforms x --platforms y
go run ./whatever --platforms x,y
```

Reply:
new docstring is
```diff
-                                                                       Usage: "comma-separated list of platforms to build, e.g. linux/amd64,linux/arm64. Defaults to build.arch in meta.json.",
+                                                                       Usage: "list of platforms to build, e.g. linux/amd64,linux/arm64. Defaults to build.arch in meta.json.",
```

lmk if it doesn't make sense